configure

```diff
@@ -71,6 +71,11 @@ find_pytest()
 	    return
 	fi
     done
+
+    if python3 -c "import pytest" ; then
+        echo "python3 -m pytest"
```

Comment:
I notice that the before loop iterates over multiple possible namings for `pytest`.  I wonder, might `python3` similarly be named differently depending on distribution? Should we not also iterate among possible `PYTHON_BINS`?

Reply:
If we are, we might need to also check that `python` actually is a version 3 and not a version 2 interpreter, so that adds a bit more work.

Reply:
I would suggest doing so, although I am uncertain how many distributions would have a version 3 python that is in `python` but not in `python3`.  This is acceptable as-is, although perhaps with a `# FIXME: find out if a Python 3 could be in python but not in python3`.